clang/lib/CIR/CodeGen/CIRGenVTables.cpp

```diff
@@ -58,7 +59,10 @@ mlir::Type CIRGenVTables::getVTableType(const VTableLayout &layout) {
 
   // FIXME(cir): should VTableLayout be encoded like we do for some
   // AST nodes?
-  return CGM.getBuilder().getAnonStructTy(tys, /*body=*/true);
+  // FIXME(cir): we should have a proper type for vtables, something that does
+  // not carry record-like declaration info.
```

Comment:
Why should we?

Reply:
Replied in https://github.com/llvm/clangir/pull/227#discussion_r1294083090.

Reply:
I'd remove this comment for now, since we're removing "NONE"